Member-Travel-Fund.md

```diff
@@ -0,0 +1,40 @@
+## Purpose
+
+To establish and administer a fund for members of the Node.js
+Foundation to travel and spread knowledge about the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be Individual Members of the Node.js Foundation.
+* Candidates must be presenting one of the following presentations:
+ * [The Node.js Foundation](https://github.com/nodejs/foundation-slides).
+
+## Process
+
+New slide decks are added to the list of approved decks on an ongoing basis.
+
+* Slide deck must be managed by the Node.js Evangelism WG.
+* Send a pull request to this file adding the slides to
+ the list above.
+* Slide deck must be approved by the TSC.
+
+Any member can apply to give an approved talk. Request may take up to a month
+for the TSC to approve or decline.
+
+* Log an issue in this repository.
+ * Include the name of the event you plan to attend.
+ * Include the location of the event and where you will be traveling from.
+ * Include the presentation you intend to give.
+ * Include the size of the stipend you wish to receive.
+  * Stipends are anywhere from $100 to $1,500 USD and vary with travel distance.
+
+The TSC will consider each proposal in its next regular meeting. While it is
+ultimately at the TSC's discretion who to approve the following considerations
+are made.
+
+* Impact: Preference given to underserved communities.
+* Cost: The larger the stipend the more critically the TSC will consider the application.
```

Comment:
I'll need to make sure that the foundation can handle a pure stipend (rather than a reimbursement which is what we've done previously). It's actually much easier to manage costs using stipends, people consistently wait until the last minute when the costs go up when they know they are being re-imbursed.

As far as "proof of purchase," we should substitute with something like "link to ticket / hotel prices."